bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,236 @@ class StylintBear:
+            you use.
+            For example: If set to ``@extends``, prefer
+            ``@extends $some-var`` instead of ``@extend $some-var``
+            or if set to ``@extend``, prefer
+            ``@extend $some-var`` over ``@extends $some-var``.
+        :param indent_size:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``0``.
+            For example: If set to ``2``, prefer
+            ``/s/smargin: 0`` over ``/s/s/smargin: 0``.
+        :param leading_zero:
+            When ``True``, prefer leading zeroes on decimal points. When
+            ``False``, disallow leading zeroes on decimal points.
+        :param max_errors:
+            Set maximum number of errors. If ``0``, all errors will
+            be shown without a limit.
+        :param max_warnings:
+            Set maximum number of warnings. If ``0``, all errors will
+            be shown without a limit.
+        :param mixed_spaces_and_tabs:
+            If a non-negative number is passed to ``indent_size``,
+            soft tabs (i.e. spaces) are assumed, and if
+            ``0`` is passed to ``indent_size``, hard tabs are assumed.
+            For example: If ``indent_size = 4`` and
+            ``mixed_spaces_and_tabs = True``, prefer
+            ``s\s\s\smargin\s0`` over ``\tmargin\s0``
+            or if ``indent_size = 0`` and
+            ``mixed_spaces_and_tabs = True``, prefer
+            ``\tmargin\s0`` over ``\s\s\s\smargin\s0``.
+        :param variable_naming_convention:
+            Enforce a particular naming convention when declaring variables.
+            Throws a warning if you don't follow the convention.
+            Supported values are ``lowercase-dash``, ``lowercase_underscore``,
+            ``camelCase``, ``BEM`` and ``False``.
+        :param strict_naming_convention:
+            By default, ``variable_naming_convention`` only looks at variable
+            names. If ``strict_naming_convention`` is set to ``True``,
+            ``variable_naming_convention`` will  also look at class and
+            ID names.
+        :param none_keyword:
+            If ``True`` check for places where ``none`` used instead of ``0``.
+            If ``False`` check for places where ``0`` could be used
+            instead of ``none``.
+        :param check_no_important_keyword:
+            If ``True``, show warning when ``!important`` is found.
+            For example: If set to ``True``, this will throw a warning::
+
+                div
+                    color red !important
+
+        :param spaces_inside_parentheses:
+            Enforce or disallow use of extra spaces inside parentheses.
+        :param placeholder:
+            Enforce extending placeholder vars when using ``@extend(s)``.
+        :param prefix_vars_with_dollar:
```

Comment:
what happens if set to `False`? Is then `$` forbidden? Or does it mean "don't care"?